SECURITY.md

```diff
@@ -6,18 +6,13 @@ vulnerabilities, as well as suggestions around hardening the software and our pr
 
 ## Reporting a suspected vulnerability
 
-It is important that suspected vulnerabilities are disclosed in a responsible way, and are not publicly disclosed until
-after they have been analysed and a fix is available.
+It is important that suspected vulnerabilities are disclosed in a responsible way, and are not publicly disclosed until after they have been analysed and a fix is available.
 
-To report a security vulnerability, send an email to keycloak-security@googlegroups.com.
+To report a security vulnerability in the Keycloak codebase, send an email to keycloak-security@googlegroups.com. Please include the version affected, provide detailed instructions on how to reproduce the issue, and include your contact information for acknowledgements. If you are reporting known CVEs related to third-party libraries used in Keycloak, please create a new GitHub issue.
```

Comment:
```suggestion
To report a security vulnerability in the Keycloak codebase, send an email to keycloak-security@googlegroups.com. Please include the version affected, provide detailed instructions on how to reproduce the issue, and include your contact information for acknowledgements. If you are reporting known CVEs related to third-party libraries used in Keycloak, please create a new GitHub issue, no need to send the email.
```

Reply:
Thanks @vmuzikar for the suggestion. We're trying to stay in line with what's on the Keycloak security page (https://www.keycloak.org/security).

If it's ok for you, let's stick with what we've got for now. If we start getting more emails like this, we can make it clear for people submitting it.

Reply:
Ok, works for me. Thanks.